src/devices/cpu/xa/xa_ops.cpp

```diff
@@ -1593,7 +1593,36 @@ void xa_cpu::mulu_byte_rd_rs(u8 rd, u8 rs) { fatalerror( "MULU.b %s, %s", m_regn
 //DIVU.b Rd, Rs
 void xa_cpu::divu_byte_rd_rs(u8 rd, u8 rs) { fatalerror( "DIVU.b %s, %s", m_regnames8[rd], m_regnames8[rs]); }
 //MULU.w Rd, Rs
-void xa_cpu::mulu_word_rd_rs(u8 rd, u8 rs) { fatalerror( "MULU.w %s, %s", m_regnames16[rd], m_regnames16[rs]); }
+void xa_cpu::mulu_word_rd_rs(u8 rd, u8 rs)
+{
+	u32 fullval = gr16(rd);
+	u16 rsval = gr16(rs);
+
+	u32 result = fullval * rsval;
+
+	if (result & 0xffff0000)
+		set_v_flag();
+	else
+		clear_v_flag();
```

Comment:
Is this also copy/paste from the divide, where a result that doesn’t fit in 16 bits is an overflow condition, or is the multiply documented to do this?  (A 16 by 16 multiply can’t give a result that doesn’t fit in 32 bits, so it can’t actually cause an overflow.)

Reply:
![muluterms](https://github.com/user-attachments/assets/d235d2f7-5aa8-46c9-99e2-e840135f6406)

Reply:
I believe we match the behavior specified in the document here

Reply:
Yep, that’s the documented behaviour.  It’s just a little unusual compared to x86, etc.